src/unless.js

```diff
@@ -0,0 +1,38 @@
+var _curry2 = require('./internal/_curry2');
+var identity = require('./identity');
+var ifElse = require('./ifElse');
+
+
+/**
+ * Creates a function that will first pass its argument to the given predicate
+ * function. If the predicate returns false, the function will return the result
+ * of calling the `whenFalse` function with the same argument. If the predicate
+ * returns true, the argument is returned as is.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @see R.ifElse, R.when
+ * @sig (a -> Boolean) -> (a -> a) -> (a -> a)
+ * @param {Function} condition A predicate function
+ * @param {Function} whenFalse A function to invoke when the `condition`
+ *                             evaluates to a falsy value.
+ * @return {Function} A new function that will process the `whenFalse` function
+ *                    depending upon the result of the `condition` predicate.
+ * @example
+ *
+ *      var coerceArray = R.unless(
+ *        R.isArrayLike,
+ *        R.ifElse(
+ *          R.has('toArray'),
+ *          R.invoker(0, 'toArray')
+ *          R.of
+ *        )
```

Comment:
I think mixing `ifElse` in the example will just serve to confuse things.  Perhaps a simpler example here?